core/src/test/java/org/frankframework/http/rest/ApiListenerTest.java

```diff
@@ -258,4 +261,10 @@ public void testGetPhysicalDestinationNameWith2Endpoints() throws Exception {
 		listener.configure();
 		assertEquals("uriPattern: [/aap, /noot]/dummy; method: PUT; consumes: JSON; produces: XML", listener.getPhysicalDestinationName());
 	}
+
+	@Test
+	void testGetEmptyBodyMessageWithHeadMethod() throws ListenerException {
+		listener.setMethod(ApiListener.HttpMethod.HEAD);
+		assertTrue(listener.processRequest(new Message("Mocked request"), new PipeLineSession()).toString().contains("[]"));
```

Comment:
What does this really test?

And should the ApiListener really return an empty message always? Why?
Important is that the ApiListenerServlet doesn't return data.

Reply:
What I wanted to test here is that we return an empty body when we call the HEAD method.
Just an empty body, no null values or so

Reply:
I'm not sure how this test actually accomplishes that, or if it's the right way to accomplish that.